pydantic/_internal/_config.py

```diff
@@ -130,7 +129,7 @@ def for_model(cls, bases: tuple[type[Any], ...], namespace: dict[str, Any], kwar
         return cls(config_new)
 
     # we don't show `__getattr__` to type checkers so missing attributes cause errors
-    if not TYPE_CHECKING:
+    if not TYPE_CHECKING:  # pragma: no branch
```

Comment:
What's the justification for the pragma here?

Reply:
coverage says this line is partially covered because the condition is never false. `no branch` (not `no cover`) says to ignore that, because `not TYPE_CHECKING` will never be false at runtime.